Assets/Example/Scripts/CountlyEntryPoint.cs

```diff
@@ -246,10 +240,10 @@ public void TestWithMultipleThreads()
     }
     public void SetLocation()
     {
-        string countryCode = "us";
-        string city = "Böston’ 墨尔本";
-        string latitude = "29.634933";
-        string longitude = "-95.220255";
+        string countryCode = "uk";
+        string city = "London";

```

Comment:
let's replace this with something that has more then just the regular latin characters. we can just put down the name London in Korean